helper/imagemin.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = function(gulp, plugins, config, name) { // eslint-disable-line func-names
+  // Return function that is executed inside of .pipe()
+  return () => {
+    const theme          = config.themes[name],
+          srcBase        = config.projectPath + theme.src,
+
+          // Look for any images in the web/images directory and minify them in place
+          imageDir       = srcBase + '/web/images/**/*',
+          minnedImageDir = srcBase + '/web/images/';
```

Comment:
As to running it on different directories, I was thinking that we could run it on this directory by default and then optionally check for an argument to specify a different folder?

I've tested this on an SVG and it did manage to minify it from 4kb to 2kb! And the repo says "Minify PNG, JPEG, GIF and SVG images with imagemin"